client/blocks/subscription-length-picker/option.jsx

```diff
@@ -0,0 +1,122 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import Badge from 'components/badge';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from '../../lib/plans/constants';
+
+let componentNumber = 0;
+
+export class SubscriptionLengthOption extends React.Component {
+	static propTypes = {
+		term: PropTypes.string.isRequired,
```

Comment:
Careful with the `isRequired` if we need any of this data to load. Alternatively we can add a guard check before we render, but it tends to be more of a pain from the caller's perspective.

Reply:
Thanks for the note! This is a sub-component that should only ever be rendered when data is available, so we are safe here.